group/serializers.py

```diff
@@ -2,7 +2,15 @@
 from rest_framework import serializers
 from django.contrib.auth.models import User
 from .models import Group
+from user.serializers import UserSerializer
 
+class GroupGetSerializer(serializers.ModelSerializer):
```

Comment:
Why do you have a different serializer for the get method?

Reply:
Do you need two serialisers? Isn't it possible to achieve with a single serializer?